test/E2E/setupE2Etests.ps1

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+using namespace System.Runtime.InteropServices
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
+
+$arch = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().ToLowerInvariant()
+if ($IsWindowsEnv) {
+    $os = "win"
+    $FUNC_EXE_NAME = "func.exe"
+} elseif ($IsMacOS) {
```

Comment:
seems like a simple:

```powershell
} else {
  # macOS or Linux
  $FUNC_EXE_NAME = "func"
}
```

Reply:
can do that but then I still need to check platform for the `$os`

Reply:
fixed this